build.gradle

```diff
@@ -425,6 +425,24 @@ task "bigtop-slaves"(type:Exec,
   commandLine command
 }
 
+task "docker-package"(type:Exec,
+        description: 'Build packages via docker build.\n' +
+                'Usage:\n  $ ./gradlew -POS=[centos-7|fedora-26|debian-9|ubuntu-16.04|opensuse-42.3] -Pprefix=TAG_PREFIX -Ptarget=GRADLE_TARGET docker-package\n' +
+                'Example:\n  $ ./gradlew -POS=debian-9 -Pprefix=1.2.1 -Ptarget=tez-pkg docker-package\n' +
```

Comment:
Any input about the target naming? Thanks!

Reply:
Valuable inputs @c0s!
I love the idea of generating targets. Speaking of the name, how about tez-pkg-ind?
Although dockerized build is our implementation detail, it can tell user what exactly the different is between tez-pkg and tez-pkg-ind. Anyhow user need to have docker installed before running any dockerized target bigtop provides. Therefore I think we should explicitly expose the info.

Reply:
I presume "ind" stands for "in docker"? Sounds good to me!
As for the docker per-requisite: perhaps we can run a sanity check (i.e. an exec and check the exit status) for docker and issue the error message if one isn't available. Thanks!

Reply:
Sounds good for pre-requisite check. Let's do it in another PR.
I'll commit this if no object.

Reply:
Cool, thanks!